html/elements/input/email.json

```diff
@@ -0,0 +1,58 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-email": {
+          "__compat": {
```

Comment:
As in the others, this should have a description:

    "description": "<code>type=\"email\"</code>"